src/iceqube/messaging/backends/inmem.py

```diff
@@ -62,21 +16,6 @@ def send(self, mailbox, message, *args, **kwargs):
         INMEM_SUPER_MAILBOX[mailbox]._put(message)
         logger.debug("SEND MAILBOX: {} MSG: {}".format(mailbox, message))
 
-    def wait(self, mailbox, timeout=None):
```

Comment:
Hmm where did you put the `wait` method? It seems like this and the abstract base class stub was removed as well.

Reply:
Yep, it wasn't used anywhere, so I got rid of it.

Reply:
Ah, gotcha.